request.go

```diff
@@ -147,7 +147,7 @@ func (r Request) SendVia(svc Service) *ResponseFuture {
 	return SendVia(r, svc)
 }
 
-// Response construct a new Response to the request, and if non-nil, encodes the given body into it.
+// Response constructs a new Response to the request, and if non-nil, encodes the given body into it.
 func (r Request) Response(body interface{}) Response {
 	rsp := NewResponse(r)
```

Comment:
This could also call `r.ResponseWithCode(body, http.StatusOK)`

Reply:
I think I'd prefer to have `Response` call `NewResponse` (which defaults to `http.StatusOK`) and `ResponseWithCode` call `NewResponseWithCode` with the given status code.

If you feel strongly about this, I'm happy to change it!